### DISCUSSION TITLE
How to unlock QuickDraw achievement?

### DISCUSSION THREAD

**Discussion by DhananjayPorwal** [community/community#34848] [Other Feature Feedback, Questions, & Ideas]
According to this [repository](https://github.com/Schweinepriester/github-profile-achievements), we have to close issue/pull-request with in 5 minutes of opening. Did exactly the same but didn't unlock the achievement. Is there is something I missed?
I tried with the 5 issues/pull-requests, but nothing happen. Did it take time to unlock the achievement?
As much I knew, achievement unlock immediately.

**Comment by radar-blip** [comment]
If the repositories were private, the issues/pull requests may not have been included. 
Check this in your settings.

![profiles](https://user-images.githubusercontent.com/39196767/193445508-d243d704-2298-4863-854c-ee070b6d90f4.png)

**Reply by DhananjayPorwal** [reply]
Bro, other unlocked achievements are visible.

**Comment by Anmol-Baranwal** [comment]
There is no special trick, just the usual tough. 

- Make sure to link the issue in the pull request and close it within 5 minutes of opening.

You can check for reference [here](https://github.com/Anmol-Baranwal)

If you have further queries, feel free to ask.

**Reply by Anmol-Baranwal** [reply]
@DhananjayPorwal 
I just wanted to check in and see if you have any remaining questions or concerns about my answer. If not, would you please mark my response as accepted?

It would be greatly appreciated and it will also help other users who might be looking for a solution to a similar issue.

**Reply by WB-15** [reply]
I have similar issue on my side.

**Reply by artiomtz** [reply]
Click the Achievements title on your profile page. If it's already in Achievements but not shown, open the achievement and toggle the eye icon for it to be shown on the profile page.

**Comment by Yutsuro** [comment]
Check "Include private contributions on my profile" in settings, and **after that,** try open/close **both** a PR and an issue.  
Those series of actions worked for me.

**Comment by larryanna00** [comment]
Good to share the solution. I just need the same. Want to unlock the achievement of this repository [perimeter led display](https://www.unilightled.com/perimeter-led-display/). I hope this will work.

**Comment by WB-15** [comment]
Hello everyone,
I would appreciate if anybody let me know the method to get pull shark achivements in git.

**Reply by WB-15** [reply]
Nobody?

**Reply by DhananjayPorwal** [reply]
Once your two-pull requests were merged, you will get it on your profile page.

**Reply by Bmars8218** [reply]
This method is right to get pull shark achievement.
Here is guide for more information.
1. Fork the repository you want to contribute to.
2. Create a new branch.
3. Make changes and commit.
4. Push changes to your forked repository.
5. Open a pull request.
6. Merge the pull request.
7. Repeat these steps to get pull shark achievement (you need to do it 2 times)

**Reply by Vivekkrdev** [reply]
there is nothing to do, just create a issue and closed it up
you will get the badge.
note: it is not recommended to do this, i am just telling you how you will get it

**Reply by Samirdevpro** [reply]
thanks it is correct info

**Comment by Vivekkrdev** [comment]
there is nothing to do, just create a issue and closed it up you will get the badge. note: it is not recommended to do this, i am just telling you how you will get it

**Reply by Samirdevpro** [reply]
thanks dude

**Reply by Vivek-kumar-dev** [reply]
thanks this is correct information
